### DISCUSSION TITLE
Error de autentificación

### DISCUSSION THREAD

**Discussion by ragamurffin** [Launch-X-Latam/MisionFrontEnd#445] [Team Posts]
Necesito ayuda, cada vez que intento entrar a la cuenta de innovaccion siempre me da este error AADSTS50012: Authentication failed.

Llevo un par de día así y no he podido avanzar con el launch x. Si alguien me puede ayudar estaría agradecido.

Aquí está más detallado
Request Id: 3158ce4f-a6f4-4447-9325-837601544e00
Correlation Id: bd4f884c-6bfd-49da-9318-f3d96c2044cd
Timestamp: 2022-10-11T11:51:45Z
Message: AADSTS50012: Authentication failed.

**Comment by FredYegres** [comment]
Me pasaba lo mismo, intenta usando una contraseña segura. A mi me funciono con una que contiene mayusculas, minusculas, numeros, caracteres especiales y mas de 15 digitos
